packages/merge-styles/etc/merge-styles.api.ts

```diff
@@ -73,4 +66,6 @@ class Stylesheet {
 // WARNING: Unsupported export: IProcessedStyleSet
 // WARNING: Unsupported export: IStyleSet
 // WARNING: Unsupported export: IFontWeight
+// WARNING: Unsupported export: InjectionMode

```

Comment:
@pgonzal when is api-extractor going to support exporting `types` and `consts`? It still does not do this. Is there a plan for this, or maybe can we help to support? `enum` exports bloat bundles and we're trying to avoid `const enum` which has caused numerous issues.